frontend/src/components/navbar/top-navbar.tsx

```diff
@@ -1,84 +1,142 @@
+                            align="center"
+                            gap="40px"
+                            style={{
+                                height: '80px',
+                                padding: '26px 0px 26px 0px',
+                            }}
+                        >
+                            {!isMobile && !user.isAdministrator && !user.isResearcher && (
+                                <>
+                                    <Wrapper>
+                                        <NavbarStyledLink to="/studies" className={({ isActive }) => isActive ? 'active' : ''}>
+                            Dashboard
+                                        </NavbarStyledLink>
+                                    </Wrapper>
+                                    <Wrapper>
+                                        <NavbarStyledLink to="/achievements" className={({ isActive }) => isActive ? 'active' : ''}>
+                            Achievements
+                                        </NavbarStyledLink>
+                                    </Wrapper>
+                                </>
+                            )}
                             <NavMenu />
-                        </Box>
-                    </Box>
+                        </Group>
+                    </Group>
                 </Container>
             </div>
             {!hideBanner && <BannersBar />}
         </nav>
-    )
-}
-
-const DesktopResearcherLinks = () => {
-    const user = useCurrentUser()
-    const isMobile = useIsMobileDevice()
-    if (isMobile || !user.isResearcher) return null
-
-    return (
-        <>
-            {/* TODO Put this back in one day when enclaves are ready */}
-            {/*<Link to="/analysis" css={{ color: 'white', textDecoration: 'none', fontWeight: 'bold' }}>*/}
-            {/*    Analysis*/}
-            {/*</Link>*/}
-        </>
-    )
-}
+    );
+};
 
-const NavMenu = () => {
+const NavMenu: React.FC = () => {
     const [opened, setOpened] = useState(false);
-    const user = useCurrentUser()
-    const isMobile = useIsMobileDevice()
+    const user = useCurrentUser();
+    const isMobile = useIsMobileDevice();
 
     const menuToggle = isMobile ? (
         <Icon icon="menu" height={30} color="white" />
     ) : (
-        <Box gap alignSelf='center' align='center' css={{
-            fontWeight: 'bold', cursor: 'pointer', whiteSpace: 'nowrap', userSelect: 'none', color: colors.white,
-        }}>
-            <span>Hi {user.firstName}</span>
+        <Flex 
+            align="center"
+            onClick={() => setOpened(!opened)}
+            style={menuToggleStyles}
+        >
+            <Text
+                c="white"
+                fz={18}
+                fw={700}
+                lh="28px"
+                ff="system-ui"
+                mr={5} 
+            >
+            Hi, {user.firstName}
+            </Text>
             <Icon icon={opened ? 'chevronUp' : 'chevronDown'} />
```

Comment:
Icon should be bigger as per design spec and should be aligned to the center. Tabler Icons should be used

Reply:
The arrow is also not visible for Researcher and Admin